tests/system/src/test/java/test/javafx/scene/lighting3D/DirectionalLightTest.java

```diff
@@ -58,8 +57,13 @@ public static void main(String[] args) throws Exception {
     public static void initFX() throws Exception {
         startupLatch = new CountDownLatch(1);
         LightingTest.light = LIGHT;
-        new Thread(() -> Application.launch(TestApp.class, (String[]) null)).start();
-        assertTrue("Timeout waiting for FX runtime to start", startupLatch.await(15, TimeUnit.SECONDS));
+
+        Util.launch(startupLatch, TestApp.class);
+    }
+
+    @AfterClass
+    public static void exit() {
+        Util.shutdown();
     }
```

Comment:
This causes the test to fail. The superclass already does a shutdown, so the subclass must not.

Reply:
yes, just noticed that, thank you.  fixed in the latest commit.